### DISCUSSION TITLE
Bed Leveling Problem with self edited config.h file

### DISCUSSION THREAD

**Discussion by ni1zzz** [mriscoc/Ender3V2S1#1206] [Q&A] [answered]
I did some adjustments in the config.h menu and now if i want to build the firmware with Auto Build Marlin Plugin i get a message `Error: can't detect a supported leveling system` . But i defined a BLtouch with linear bed leveling and all pins are right configured. I checked the config file several times and everything should work properly.

**Comment by mriscoc** [comment, marked as answer]
Only the leveling systems defined by the Configurator are supported by the PROUI extensions, for other leveling systems, PROUI_EX must be disabled.

**Reply by ni1zzz** [reply]
How can i disable it? I found no code to disable it.

**Reply by mriscoc** [reply]
Look at Configuration.h file

**Reply by ni1zzz** [reply]
Even with proui disabled it displays the same error

**Reply by mriscoc** [reply]
Yes, recently versions need that you use only the predefined leveling modes in the Configurator.
